go-controller/pkg/ovn/copp_test.go

```diff
@@ -128,7 +128,8 @@ func TestEnsureDefaultCOPP(t *testing.T) {
 			desc:      "cleans multiple empty-name default COPPs and adds named COPP",
 			expectErr: false,
 			initialNbdb: libovsdbtest.TestSetup{
-				NBData: multipleEmptyCOPPNameNBData,
+				IgnoreConstraints: true,
```

Comment:
hmmm can't wrap my head around this one.

how can there multiple empty-name default COPPs in a real DB? 

I guess the name index was added in an schema upgrade, but wouldn't the DB be forced to handle that in some way before we even notice it client-side?

Or is this a test targeting an scenario where we were mopping up the DB before that schema upgrade actually happened?

Reply:
@jcaamano Yes, that was the issue. Before OVN 22.09 COPP objects had no name and no index on name. So here we're testing that we can fix the DB on upgrade.